templates/itemList.html

```diff
@@ -0,0 +1,50 @@
+{% macro itemList(items) %}
+<ul class="list-group mb-4">
+    {% for item in items %}
+        {{ notStartedItem(item) if item.status == "Not Started" else completedItem(item) }}
+    {% endfor %}
+</ul>
+{% endmacro %}
+
+{% macro notStartedItem(item) %}
+<li class="list-group-item list-group-item-info">
```

Comment:
I'd probably indent these one level in (to show they are contained within the macro)